test/integration-tests/UniversalRouterIntegration.test.ts

```diff
@@ -0,0 +1,350 @@
+import { UniversalRouter, Permit2, ERC721 } from '../../typechain';
```

Comment:
As I understand `UniversalRouterIntegration.test.ts` was `UniversalRouter.test.ts` before, can you please restore it? And move `UniversalRouter.test.ts` to the `foundry-tests` directory, to keep the structure as it was